javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -729,12 +785,16 @@ private predicate storeStep(
  * Holds if `f` may `read` property `prop` of parameter `parm`.
  */
 private predicate parameterPropRead(
-  Function f, DataFlow::Node invk, DataFlow::Node arg, string prop, DataFlow::PropRead read,
+  Function f, DataFlow::Node invk, DataFlow::Node arg, string prop, DataFlow::Node read,
   DataFlow::Configuration cfg
 ) {
-  exists(DataFlow::SourceNode parm |
+  exists(DataFlow::Node parm |
     callInputStep(f, invk, arg, parm, cfg) and
-    read = parm.getAPropertyRead(prop)
+    (
+      read = parm.(DataFlow::SourceNode).getAPropertyRead(prop)
+      or
+      isAdditionalLoadStep(parm, read, prop, cfg)
```

Comment:
```suggestion
      exists(DataFlow::Node use | parm.flowsTo(use) |
        isAdditionalLoadStep(use, read, prop, cfg)
      )
```

`getAPropertyRead` has an implicit `flowsTo/getALocalSource` call which we seem to be missing in the `isAdditionalLoadStep` disjunct.